tests/functional/encryption/test_encryption_keyrotation.py

```
@@ -70,73 +70,96 @@ def test_osd_keyrotation(self):
                     )
-                log.info(f"Keyrotation is happend for device {device}")
+                logger.debug(f"Keyrotation happened for device {device}")
             return True
 
-        # with pytest.raises(UnexpectedBehaviour):
         try:
             compare_old_with_new_keys()
         except UnexpectedBehaviour:
-            log.error("Key rotation is Not happend after schedule is passed. ")
+            logger.exception("Key rotation did not happen after schedule passed")
             assert False
```

Ruff (0.15.15)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)